Cognite.Jetfire.Cli/ListTransforms/ListCommand.cs

```diff
@@ -29,34 +29,41 @@ async Task Handle(string cluster)
             {
                 var list = await client.TransformConfigList(new System.Threading.CancellationToken());
 
+                // Sort on ID descending (and thus created at time)
+                list.Sort((x, y) => x.Id.CompareTo(y.Id));
+
                 // Width of columns (initialized with length of col title)
                 var IdLength = 2;
                 var ExternalIdLength = 11;
                 var NameLength = 4;
                 var ScheduleLength = 8;
+                var LastRunLength = 7;
 
                 // Calculate col width
-                List<Tuple<string, string, string, string>> rows = new List<Tuple<string, string, string, string>>();
+                List<Tuple<string, string, string, string, string>> rows = new List<Tuple<string, string, string, string, string>>();
```

Comment:
```suggestion
                var rows = new List<(string id, string extid, string name, string sched, string lastRun)>();
```

Reply:
Why not use the C# value tuple, it seems much more readable to me? (or do we have to be compatible with some old C# versions?)

Reply:
That's much nicer! 😄 I was just not aware of this when I wrote it (not super-familiar with C#, and googling 'C# tuples' yielded the solution you see here).

How new is that syntax? How far back would we cut support by switching over?

Reply:
It's not that new. I just checked and since we already require at least .NET Core 2.0, it's not a problem (I think that when targeting the old windows-only .NET, you might have to install some NuGet package or so, but that's not our case)